stripe/stripe-terminal-ios
# [Example app] Use xcframeworks from Carthage for deps and StripeTerminal.xcframework

# Summary
This is to fix things up so the Example project works without any extra tinkering with Xcode 12.4. tl;dr, use xcframeworks.

This is a noisy change :/...human changes are:
* add `--use-xcframeworks` to carthage update call in setup.sh and then rm -rf Carthage and run setup.sh with carthage 0.37.0
* add the StripeTerminal.xcframework, as downloaded from the 1.4.0 release, to the repo
* edit the xcodeproj:
  * remove the .frameworks and add .xcframeworks, set to embed & sign.
  * remove `-FCarthage...` from the framework search path (shouldn't be needed)

# Test plan:
* confirm Xcode 12.4 builds & runs (tested x86 sim and arm device - _still known that arm sim isn't included_)
* confirm Xcode 12.1 errors about the prebuilt dependencies using swift 5.3.2 but the project is building with 5.3
* re-run ./setup.sh with 12.1 as the xcode-select Xcode
* confirm after a clean and rebuild 12.1 is happy
